data-transforms/redaction/transform.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/redpanda-data/redpanda/src/go/transform-sdk"
```

Comment:
This is the old module. It should be the following now:


```
"github.com/redpanda-data/redpanda/src/transform-sdk/go/transform"
```

I think this just made it into the latest RC